### ISSUE TITLE
Allow to run crab commands with request name only, without requiring a working directory

### ISSUE THREAD

**Issue by matz-e** [dmwm/CRABClient#4607]
I've had a case here in the office where someone wanted to kill their crab task **after** deleting the corresponding working directory.

Looking at the crab directories, it is actually possible to rig up a `.requestcache` that seems to be able to access the CRAB task without the original working directory. I'm not sure that the kill command worked right, since the jobs were pending and got retried after the kill command was issued.

Why can't we make crab commands accept a request name instead of a working directory, at least for some commands (status, kill)?

**Comment by belforte** [comment]
this should work:
crab remake --task taskname
crab kill -d <the directory created by above command>

**Comment by belforte** [comment]
as to the question... I do not know, I ask same in the past and was told that it requires too much work.
But I can't think of any deep reason why .requestcache is needed. Once the task is submitted all information that may ever be possibly desired is on the server.  Maybe the local directory is there in order to store the ISB (...default.tgz) ... even if it is deleted from user file cache ?

**Comment by matz-e** [comment]
Thanks, that'll probably do. Having the ability to just use the request name would be nice, but if it's too much work… there are bigger fish to fry.

**Comment by mmascher** [comment]
IMHO the use case "I accidentally deleted the taskdir" is covered with the `--remake` command, there is no real need to introduce an option `--task` to the commands. Remake take care of setting up the necessary directory structure with the logfile put in the right place and so on.

I also found a branch where I did a first implementation of that (https://github.com/mmascher/CRABClient/commit/6de881512d3e89825efd37ef8a837c06d75a54d9), but my problem with that was that we were introducing more code to account for an already covered use case. And as usual there was the problem of where put the logfiles in that case, `uploadlog` not working well without a project dir and in general comamnds assuming they run in a dedicated directory.

So it's possible, but remake is already there and its code is more mantainable long term.
